packages/widget/src/types/events.ts

```diff
@@ -6,6 +6,7 @@ export enum WidgetEvent {
   RouteExecutionCompleted = 'routeExecutionCompleted',
   RouteExecutionFailed = 'routeExecutionFailed',
   RouteHighValueLoss = 'routeHighValueLoss',
+  RoutesAvailable = 'routesAvailable',
```

Comment:
If we are going to return the routes array, maybe it would be better to call it AvailableRoutes. In that case, we can assume that there can be no available routes, while with RoutesAvailable it looks like we always return only available ones 😄

Reply:
Nice! It´s now renamed to "AvailableRoutes"